README.md

```diff
@@ -81,12 +73,8 @@ Now let's *import* the _managed cluster_ to the _hub_ by deploying the [Klusterl
 We're now going to switch to the KinD cluster named `cluster` and configure the `Klusterlet` operator and supporting `Klusterlet` operand.
 
 ```bash
-
-# Install the Operator Lifecycle Manager if not present
-$ curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.18.1/install.sh | bash -s v0.18.1
-
-# Configure the Klusterlet operator
-$ kubectl create -f https://operatorhub.io/install/klusterlet.yaml
+$ kubectl config use kind-cluster
```

Comment:
@qiujian16 I used this method because it's what is officially captured on operatorhub.io. If we think users should leverage what is in `registration-operator`, let's start by updating the operatorhub.io instructions.

Reply:
I think either way is fine. Some feedback says that they would want to install the operator without olm as the pre-requirement. I think that is what we can provide directly from registration-operator or clusteradm cli.